container/engine.py

```diff
@@ -528,6 +530,17 @@ def cmdrun_version(base_path, engine_name, debug=False, **kwargs):
         engine_obj = load_engine(**engine_args)
         engine_obj.print_version_info()
 
+def cmdrun_starbuck(base_path, alternate_filename=None, **kwargs):
```

Comment:
Let's solicit at commandline the project name as well, defaulting to the name of the directory that the Dockerfile is found in.